synse/websocket_test.go

```diff
@@ -180,6 +180,45 @@ func TestWebSocketClientV3_Version_200(t *testing.T) {
 	assert.NotNil(t, resp)
 	assert.NoError(t, err)
 	assert.Equal(t, expected, resp)
+
+	err = client.Close()
+	assert.NoError(t, err)
+}
+
+func TestWebSocketClientV3_Version_500(t *testing.T) {
+	in := `
+{
+   "id":1,
+   "event":"response/error",
+   "data":{
+      "http_code":500,
+      "error_id":0,
```

Comment:
I believe `error_id` is going to be removed from the error response scheme in v3. If it still exists somewhere in the spec, thats an issue with the spec.

This should be the correct scheme: https://github.com/vapor-ware/synse-server/blob/v3/staging/proposals/v3/api.md#scheme 

Really the only changes are:
- no more `error_id` field
- timestamp is in RFC3339 (second resolution) instead of RFC3339Nano (nanosecond resolution)